src/Storages/PartitionStrategy.cpp

```diff
@@ -0,0 +1,150 @@
+#include <Common/re2.h>
+#include <Storages/PartitionStrategy.h>
+#include <Parsers/ASTFunction.h>
+#include <Parsers/ASTLiteral.h>
+#include <Interpreters/TreeRewriter.h>
+#include <Interpreters/ExpressionAnalyzer.h>
+#include <Storages/PartitionedSink.h>
+#include <Functions/generateSnowflakeID.h>
+
+#include "Interpreters/ExternalLoaderDictionaryStorageConfigRepository.h"
```

Comment:
I am using a new version of my IDE and I suspect it is, for some freaking odd reason, adding some random include statements